nautobot/core/templates/inc/nav_menu.html

```diff
@@ -2,6 +2,59 @@
 {% load registry %}
 {% load helpers %}
 {% registry %}
+
+<style>
```

Comment:
Should/could any of this styling be added to the `nautobot/project-static/css/base.css` stylesheet instead of being inline in this file? I know we do a fair amount of styling of the nav menus in that file already and am concerned about possible conflicts between the two.

Reply:
Happy to move the CSS into the `base.css` file.  Will make those updates.